zhaquirks/philips/zhadimmablelight.py

```diff
@@ -39,11 +39,14 @@ class ZHADimmableLight(CustomDevice):
             (PHILIPS, "LWA007"),
             (PHILIPS, "LWO001"),
             (PHILIPS, "LWO003"),
+            (PHILIPS, "LWU001"),
             (PHILIPS, "LWV001"),
             (SIGNIFY, "LWA004"),
             (SIGNIFY, "LWA005"),
+            (SIGNIFY, "LWA007"),
             (SIGNIFY, "LWO001"),
             (SIGNIFY, "LWO003"),
+            (SIGNIFY, "LWU001"),
```

Comment:
are these changes intentional? they aren't related to the device you are adding support for...

Reply:
You are right. I pull the dev. My mistake. But I think this commit under the right PR is OK. I'm working around ZB philips OTA too.

Reply:
An action is needed?

Reply:
I'm closing this PR as it seems to make trouble. I will publish a new one without issues in one commit